grafana-plugin/src/components/ManualAlertGroup/ManualAlertGroup.tsx

```diff
@@ -26,55 +43,137 @@ const ManualAlertGroup: FC<ManualAlertGroupProps> = (props) => {
   const store = useStore();
   const [userResponders, setUserResponders] = useState([]);
   const [scheduleResponders, setScheduleResponders] = useState([]);
-  const { onHide, onCreate } = props;
+  const { onHide, onCreate, alertReceiveChannelStore } = props;
   const data = { team: store.userStore.currentUser?.current_team };
 
+  const [selectedTeamId, setSelectedTeam] = useState<GrafanaTeam['id']>();
+  const [selectedTeamDirectPaging, setSelectedTeamDirectPaging] = useState<AlertReceiveChannel>();
+  const [directPagingLoading, setdirectPagingLoading] = useState<boolean>();
+
   const handleFormSubmit = async (data) => {
     store.directPagingStore
       .createManualAlertRule(prepareForUpdate(userResponders, scheduleResponders, data))
-      .then(({ alert_group_id: id }: { alert_group_id: Alert['pk'] }) => {
+      .then(({ alert_group_id: id }: { alert_group_id: AlertType['pk'] }) => {
         onCreate(id);
       })
       .finally(() => {
         onHide();
       });
   };
 
+  const onUpdateSelectedTeam = async (selectedTeamId: GrafanaTeam['id']) => {
+    setdirectPagingLoading(true);
+    setSelectedTeamDirectPaging(null);
+    setSelectedTeam(selectedTeamId);
+    await alertReceiveChannelStore.updateItems({ team: selectedTeamId, integration: 'direct_paging' });
+    const directPagingAlertReceiveChannel =
+      alertReceiveChannelStore.getSearchResult() && alertReceiveChannelStore.getSearchResult()[0];
+    if (directPagingAlertReceiveChannel) {
+      setSelectedTeamDirectPaging(directPagingAlertReceiveChannel);
+    }
+    setdirectPagingLoading(false);
+  };
+
   const onUpdateEscalationVariants = useCallback(
     (value) => {
       setUserResponders(value.userResponders);
-
       setScheduleResponders(value.scheduleResponders);
     },
     [userResponders, scheduleResponders]
   );
 
+  const warningTitle = (
+    <>
+      <TeamName team={store.grafanaTeamStore.items[selectedTeamId]} />{' '}
+      <Text>team doesn't have the the Direct Paging integration yet</Text>
+    </>
+  );
+
+  const DirectPagingIntegrationVariants = observer(({ selectedTeamId, selectedTeamDirectPaging }) => {
```

Comment:
seems you don't need to wrap it in `observer` because `DirectPagingIntegrationVariants` component doesn't use `store` inside